release-content/0.14/release-notes/13347_Add_on_unimplemented_Diagnostics_to_Most_Public_Traits.md

```diff
@@ -1 +1,70 @@
-TODO
+Bevy takes full advantage of the powerful type system Rust provides, but with that power can often come confusion when even minor mistakes are made.
+
+```rust
+use bevy::prelude::*;
+
+struct MyResource;
+
+fn main() {
+    App::new()
+        .insert_resource(MyResource)
+        .run();
+}
+```
+
+Running the above will produce a compiler error, let's see why...
+
+<details>
+<summary>Click to expand...</summary>
+
+```output
```

Comment:
Is `output` a valid language in Zola? What do we usually use for compiler outputs?

Reply:
Excellent question! I based the choice of `output` on an unrelated GitHub discussion. Looking at [these docs](https://www.getzola.org/documentation/content/syntax-highlighting/) indicates I should _probably_ just use backticks without a language specified to indicate it is plain-text. I'll make that change but leave this unresolved just in case someone else knows better.